src/System.IO.Compression.Brotli/README.md

```diff
@@ -0,0 +1,214 @@
+﻿# System.IO.Compression.Brotli
+
+## Introduction
+
+Brotli is a generic-purpose lossless compression algorithm that compresses data
```

Comment:
That was my reaction too. However, this text was lifted from the official Google Brotli repo, so I decided to ignore it.